zulip/zulip/__init__.py

```diff
@@ -1071,6 +1071,20 @@ def get_user_by_id(self, user_id, request=None):
             request=request,
         )
 
+    def deactivate_user_by_id(self, user_id):
+        # type: (int) -> Dict[str, Any]
+
+        '''
+            Example usage:
+
+            >>> client.delete_user_by_id(8)
```

Comment:
Why is this called `delete_user_by_id` when the function name is `deactivate_user_by_id`?

Reply:
Yeah, that's a typo. I'll fix that.